src/SFA.DAS.Assessor.Functions.ExternalApis/Assessor/AssessorServiceApiClient.cs

```diff
@@ -0,0 +1,173 @@
+            }
+        }
+
+        public async Task SetAssessorSetting(string name, string value)
+        {
+            using (var request = new HttpRequestMessage(HttpMethod.Put, $"api/v1/assessor-setting/{name}/{value}"))
+            {
+                await PostPutRequest(request);
+            }
+        }
+
+        public async Task<string> GetAssessorSetting(string name)
+        {
+            using (var request = new HttpRequestMessage(HttpMethod.Get, $"api/v1/assessor-setting/{name}"))
+            {
+                return await GetAsync<string>(request);
+            }
+        }
+
+        public async Task<ImportLearnerDetailResponse> ImportLearnerDetails(ImportLearnerDetailRequest importLearnerDetailRequest)
+        {
+            using (var request = new HttpRequestMessage(HttpMethod.Post, "api/v1/learnerdetails/import"))
+            {
+                return await PostPutRequestWithResponse<ImportLearnerDetailRequest, ImportLearnerDetailResponse>(request,
+                    importLearnerDetailRequest);
+            }
+        }
+
+        public string BaseAddress()
+        {
+            return Client.BaseAddress.ToString();
+        }
+
+        public async Task<BatchLogResponse> CreateBatchLog(CreateBatchLogRequest createBatchLogRequest)
+        {
+            var responseMessage = await Client.PostAsJsonAsync($"/api/v1/batches", createBatchLogRequest);
+            return await responseMessage.Content.ReadAsAsync<BatchLogResponse>();
+        }
+
+        public async Task ChangeStatusToPrinted(int batchNumber, IEnumerable<CertificateResponse> responses)
+        {
+            // the certificate printed status be will updated in chunks to stay within the WAF message size limits
+            const int chunkSize = 100;
+
+            var certificateStatuses = responses.Select(
+                q => new CertificateStatus
+                {
+                    CertificateReference = q.CertificateReference,
+                    Status = Assessor.Constants.CertificateStatus.Printed
+                }).ToList();
+
+            foreach (var certificateStatusesChunk in certificateStatuses.ChunkBy(chunkSize))
+            {
+                var updateCertificatesBatchToIndicatePrintedRequest = new UpdateCertificatesBatchToIndicatePrintedRequest
+                {
+                    BatchNumber = batchNumber,
+                    CertificateStatuses = certificateStatusesChunk
+                };
+
+                await Client.PutAsJsonAsync($"/api/v1/certificates/{batchNumber}", updateCertificatesBatchToIndicatePrintedRequest);
+            }
+        }
+
+        public async Task CompleteSchedule(Guid scheduleRunId)
+        {
+            await Client.PostAsync($"/api/v1/schedule?scheduleRunId={scheduleRunId}", null);
+        }
+
+        public async Task<IEnumerable<CertificateResponse>> GetCertificatesToBePrinted()
+        {
+            var response = await Client.GetAsync("/api/v1/certificates/tobeprinted");
```

Comment:
As above about using the client the base class has method for accessing the response